docs/platform-setup.md

```diff
@@ -1,12 +1,12 @@
-# Setup of the platform Jenkins X Team
+# Setup Of The Platform Jenkins X Team
```

Comment:
Sorry for being super picky but usually, and according to https://capitalizemytitle.com/, we don't capitalize "of", "the", ...

Reply:
you are wright

Reply:
changes applied